modules/gdscript/gd_functions.cpp

```diff
@@ -381,6 +384,24 @@ void GDFunctions::call(Function p_func,const Variant **p_args,int p_arg_count,Va
 			VALIDATE_ARG_NUM(0);
 			r_ret=Math::db2linear(*p_args[0]);
 		} break;
+		case MATH_RAD2VEC: {
+			VALIDATE_ARG_COUNT(1);
+			VALIDATE_ARG_NUM(0);
+			float v2x, v2y;
+			v2x = Math::cos(*p_args[0]);
+			v2y = Math::sin(*p_args[0]);
+			Vector2 v(v2x, v2y);
+			r_ret = v;
```

Comment:
I used extra lines for clarity, but the compressed version does match the style better. Changed.